docs/docs/guides/tracking/tracing.mdx

```diff
@@ -127,30 +124,53 @@ with weave.attributes({'env': 'production'}):
 ### 3. Manual Call Tracking
 
 You can also manually create Calls using the API directly.
-* Start a call: [POST `/call/start`](../../reference/service-api/call-start-call-start-post.api.mdx)
-* End a call: [POST `/call/end`](../../reference/service-api/call-end-call-end-post.api.mdx)
 
-For example:
+<Tabs groupId="client-layer">
+    <TabItem value="python_sdk" label="Python">
+    ```python showLineNumbers
+    import weave
+
+    # Initialize Weave Tracing
+    weave.init('intro-example')
+
+    def my_function(name: str):
+        # Start a call
+        call = weave.create_call(op="my_function", inputs={"name": name})
+
+        # ... your function code ...
+
+        # End a call
+        weave.finish_call(call, output="Hello, World!")
+
+    # Call your function
+    print(my_function.call("World"))
```

Comment:
Should this just be `my_function("World")`

Reply:
good catch -- copied too much of the existing code 🤦 

while we're at it -- any reason to present the other examples on this page this way?

Reply:
Sorry, not sure what you mean

Reply:
<img width="992" alt="image" src="https://github.com/user-attachments/assets/3550f236-7cdb-4407-82f2-4c7b2fb6b7cd">

^ here's the example we have for using @weave.op. Not sure why we want to highlight that you can do .call here. Should we just simplify?

Reply:
Oh I see - yes I agree, no reason to use `.call` here